tasks/lib/jscs.js

```diff
@@ -58,16 +58,18 @@ exports.init = function( grunt ) {
     }
 
     /**
-     * @see Checker#checkPath
+     * @see Checker#checkPath and Checker#fixPath
      */
-    JSCS.prototype.check = function( path ) {
-        var checkPath = this.checker.checkPath( path );
+    JSCS.prototype.checkOrFix = function( path ) {
```

Comment:
Oh, okay, how about we add a method that return boolean value telling us if this option is enabled.

Then before the https://github.com/jscs-dev/grunt-jscs/pull/84/files#diff-7875685af27dacb398c11053ef46c5d0L21, we could call that method and depending of the return value, execute either, `check` or `fix` method.

How is that sound?